tornado-benchmarks/src/main/java/uk/ac/manchester/tornado/benchmarks/saxpy/SaxpyTornado.java

```diff
@@ -100,11 +98,4 @@ public boolean validate(TornadoDevice device) {
         return ulp < MAX_ULP;
     }
 
-    public void printSummary() {
-        if (isValid()) {
-            System.out.printf("id=%s, elapsed=%f, per iteration=%f\n", TornadoRuntime.getProperty("benchmark.device"), getElapsed(), getElapsedPerIteration());
-        } else {
-            System.out.printf("id=%s produced invalid result\n", TornadoRuntime.getProperty("benchmark.device"));
-        }
-    }
```

Comment:
Can you please clarify why this part is now deleted?

Reply:
is it not useful anymore? and do we dump the information from a different point?

Reply:
We never call these methods, so we can delete them,

Reply:
thanks